app/views/general_feedback/new.html.haml

```diff
@@ -23,6 +23,15 @@
                 hint: t('general_feedback.visit_purpose_hint_text').html_safe,
                 input_html: { class: 'form-control form-control-4-4', rows: '5' }
 
-      = render partial: '/feedback/fields', locals: { form: f }
+      .govuk-form-group.general-feedback-form-group
+        %fieldset.govuk-fieldset
+          %legend.govuk-fieldset__legend.govuk-fieldset__legend--m
+            %h2.govuk-fieldset__heading
+              = t('feedback.comment_label')
+
+      = f.input :comment, as: :text,
```

Comment:
I think the indentation here is off, which is causing the text box to be placed outside the fieldset.

You can read up on how fieldsets should be used here:
https://design-system.service.gov.uk/components/fieldset/